Laravel 5.7.10 changes (#135)
tightenco/collect: readme.md

Lovingly split by Matt Stauffer for [Tighten Co.](https://tighten.co/), with a kick in the butt to finally do it from [@assertchris](https://github.com/assertchris).

## Install

With [Composer](https://getcomposer.org):

```bash
composer require tightenco/collect
```

> ## Upgrade

Reviewer: Please mention it defaults to the latest version of 5.7 but you can also pass a specific version

Author: Good call, I've just added it to the readme.

To upgrade Collect, simply run `./upgrade.sh` from the root directory. This will pull in the recent Laravel changes, update the Collect package and run the unit tests.

> The upgrade script requires the use of `wget`. To install [homebrew](https://brew.sh), and run `brew install wget`

## Test

> **Due to a [dependency on Carbon](https://github.com/tightenco/collect/commit/4afe1fcb40f1c10e399730562c2c7ca36c6fba01), tests need to be ran after `./upgrade.sh`.**

Reviewer: ```suggestion
**Due to a [dependency on Carbon](https://github.com/tightenco/collect/commit/4afe1fcb40f1c10e399730562c2c7ca36c6fba01), tests won't pass until you've run `./upgrade.sh` at least once locally.**
```

Lovingly split by Matt Stauffer for [Tighten Co.](https://tighten.co/), with a kick in the butt to finally do it from [@assertchris](https://github.com/assertchris).

## Install

With [Composer](https://getcomposer.org):

```bash
composer require tightenco/collect
```

> ## Upgrade

Reviewer: Maybe rename "Upgrade" to "Development" ? or something like that?

Author: I think Development makes a lot more sense, because someone could be tasks with something other than upgrading Collect. The first sentence now tackles upgrading. Thank you for the re-write session.

> To upgrade Collect, simply run `./upgrade.sh` from the root directory. By default this will pull in the most recent Laravel 5.7 changes, update the Collect package and run the unit tests. The upgrade script accepts a Laravel version, so a specific version can be passed in.

Reviewer: ```suggestion
If you are a developer working on Collect and you're tasked with upgrading it to mirror a new version of Laravel,  run `./upgrade.sh` from the root directory. You can pass a parameter to target a specific Laravel version (e.g. `./upgrade.sh 5.7.10`) or, if you don't pass a parameter, the script will find the latest tagged release and run against that.

The upgrader will pull down the appropriate source and test files for the specified version of Laravel and then run the tests.
```

```bash

> ./updrade.sh 5.7.10

Reviewer: ```suggestion
./upgrade.sh
# or
./upgrade.sh 5.7.10
```

```

> 

Reviewer: Delete extra line
